src/components/Event.jsx

```diff
@@ -33,8 +33,8 @@ class EventBox extends React.Component {
 
   style (props) {
 
-    var length = `${props.data._draw_length * 100}%`
-    var position = `${props.data._draw_position * 100}%`
+    var length = `${props.data._length * 100}%`
+    var position = `${props.data._position * 100}%`
```

Comment:
You are still using `var` here.
Also, it is not very clear whether there is difference between `this.props` and `props` parameter; if they are the same, choose one option. If you are passing in different props, add a comment and expand the object, e.g. `{data}`.